src/providers/ogr/qgsgeopackagedataitems.cpp

```diff
@@ -525,9 +525,23 @@ bool QgsGeoPackageAbstractLayerItem::deleteLayer()
   }
   else
   {
-    QMessageBox::information( nullptr, tr( "Delete Layer" ), tr( "Layer <b>%1</b> deleted successfully." ).arg( mName ) );
     if ( mParent )
+    {
+      QgsGeoPackageConnectionItem *connectionParentItem = qobject_cast<QgsGeoPackageConnectionItem *>( mParent );
+      if ( connectionParentItem )
+      {
+        if ( QMessageBox::question( nullptr, QObject::tr( "Delete Layer" ), QObject::tr( "The layer <b>%1</b> deleted successfully."
```

Comment:
Should this be equal to the other message?

```suggestion
        if ( QMessageBox::question( nullptr, QObject::tr( "Delete Layer" ), QObject::tr( "Layer <b>%1</b> deleted successfully."
```